src/sentry/static/sentry/app/views/alerts/details/header.tsx

```diff
@@ -141,23 +141,23 @@ export default class DetailsHeader extends React.Component<Props> {
                   </Projects>
                 )}
               </ItemValue>
-              {isErrorDataset && stats && (
+              {isErrorDataset && (
                 <ItemValue>
-                  <Count value={stats.uniqueUsers} />
+                  {stats ? <Count value={stats.uniqueUsers} /> : t('Loading')}
                 </ItemValue>
               )}
-              {isErrorDataset && stats && (
+              {isErrorDataset && (
                 <ItemValue>
-                  <Count value={stats.totalEvents} />
+                  {stats ? <Count value={stats.totalEvents} /> : t('Loading')}
```

Comment:
Can we use `<Placeholder>` instead of the string?

Reply:
![image](https://user-images.githubusercontent.com/1400464/91777096-c7a69080-eba3-11ea-9ec3-8402836f267a.png)